salt/utils/templates.py

```diff
@@ -346,6 +348,30 @@ def render_jinja_tmpl(tmplstr, context, tmplpath=None):
 
     jinja_env.filters['strftime'] = salt.utils.date_format
     jinja_env.filters['sequence'] = salt.utils.jinja.ensure_sequence_filter
+    jinja_env.filters['http_query'] = salt.utils.http.query
+    jinja_env.filters['to_bool'] = salt.utils.jinja.to_bool
+    jinja_env.filters['default'] = salt.utils.jinja.default
+    jinja_env.filters['quote'] = salt.utils.jinja.quote
+    jinja_env.filters['escape'] = salt.utils.jinja.regex_escape
+    jinja_env.filters['regex_search'] = salt.utils.jinja.regex_search
+    jinja_env.filters['regex_replace'] = salt.utils.jinja.regex_replace
+    jinja_env.filters['uuid'] = salt.utils.jinja.uuid_
+    jinja_env.filters['unique'] = salt.utils.jinja.unique
+    jinja_env.filters['min'] = salt.utils.jinja.lst_min
+    jinja_env.filters['max'] = salt.utils.jinja.lst_max
+    jinja_env.filters['sum'] = salt.utils.jinja.lst_sum
```

Comment:
I just realised that some filters such ```sum``` have been introduced in the recent version of Jinja: http://jinja.pocoo.org/docs/dev/templates/#sum

Will remove duplicated tomorrow.